sql/core/src/test/scala/org/apache/spark/sql/DataFrameAggregateSuite.scala

```diff
@@ -36,6 +36,8 @@ case class Fact(date: Int, hour: Int, minute: Int, room_name: String, temp: Doub
 class DataFrameAggregateSuite extends QueryTest with SharedSQLContext {
   import testImplicits._
 
+  val absTol = 1e-8
```

Comment:
Since we always use `1e-8` for `checkAggregatesWithTol` in this test suite, how about setting this value as default in the function argument?

Reply:
we cannot because this would cause a compile error since there are two overloaded functions:
```
in class QueryTest, multiple overloaded alternatives of checkAggregatesWithTol define default arguments
```

Reply:
Ah, ok.